packages/unit-jest/src/prompts.js

```diff
@@ -25,6 +25,10 @@ module.exports = function () {
           name: 'TypeScript support',
           value: 'typescript',
         },
+        {
+          name: 'SWC for TypeScript (experimental)',
```

Comment:
I'd use "Use SWC instead of ts-jest to transpile TypeScript (experimental, at least 20x faster transpilation, see <add-relevant-link>)"
It's way longer, but provides more context about what SWC project is and why it's useful